Automattic/jetpack
# Add new Recipe block to Jetpack

#### Changes proposed in this Pull Request:
This PR adds a new Gutenberg block with contained child blocks that output a semantic and easy to use Recipe.

Designs for this block can be found [here](https://www.figma.com/file/aEzvgjhnGwWSNMQNOYQ7Pa/Recipies-Block?node-id=0%3A1)

As this is a draft PR, there are a few items yet to be completed:

- [x] Semantic Schema markup
- [x] Provide color options for users to adjust the associated bulleted list output by the Steps component

#### Does this pull request change what data or activity we track or use?

My PR does not change any data or activity tracking.

#### Testing instructions:

- Create a new post or page, or navigate to an existing post or page
- Add the Recipe block in whichever manner you choose (by typing `/` in the editor, or using the Block Inserter).
- Select the "Recipe" block
- Using the provided block template add your preferred level of detail for the recipe. The block should be able to handle most levels of detail, or lack thereof.
- The top level Reicpe block, and child "Steps" block have block attributes that provide further detail or customization options. Feel free to adjust these as you wish.

The aim is for this block to be fairly structured so folks don't have to think about layout, so hopefully the block template provides clear indications for what is available within the block itself.
